src/Downloader.js

```diff
@@ -36,8 +36,10 @@ export default class Downloader extends Step {
   }
 
   getPackage(pkg, version, { shasum, tarball }) {
+    const tarballUri = tarball ? tarball.replace(/https:\/\/registry.npmjs.org/i, 'http://registry.npmjs.org') : tarball;
```

Comment:
I don't think this makes sense.

Reply:
@luisdavim What doesn't make sense about it?

Reply:
forcing unsafe practices doesn't make sense, it is for a good reason that the url in the metadata returned by npmjs.org used https even when you use plain http to fetch the metadata.

Reply:
I disagree - I believe we should make the whole thing use https by default and close off that vulnerability.